package.json

```diff
@@ -63,17 +67,5 @@
     "webpack": "5.75.0",
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.7.4"
-  },
-  "consolePlugin": {
-    "name": "console-plugin-template",
```

Comment:
Oh, I missed that. I'll check it. Thanks!

Reply:
Indeed, `start-console.sh` reads `${npm_package_consolePlugin_name}` provided by Yarn.

The main reason for extracting dynamic plugin metadata to a `.ts` file is type checking and other code related benefits.

@spadgett Do you think it's OK to treat `name` in `package.json` as dynamic plugin metadata name? If so, it could be read as `${npm_package_name}` in `start-console.sh` script.

I'd generally recommend against relying on `${npm_package_xx}` variables, IMO it's better to pass that information explicitly e.g. in `package.json`
```jsonc
"start-console": "PLUGIN_NAME=console-plugin-template ./start-console.sh",
// or "PLUGIN_NAME=$(resolve-plugin-name-here) ./start-console.sh",
```

Reply:
Update - I think it's possible to resolve the plugin name from `plugin-metadata.ts` via ts-node execution.

This way, we can keep the metadata in a `.ts` file while having the ability to pass the plugin name to any scripts.

Reply:
@spadgett I've updated the PR to resolve plugin name via `ts-node` in `start-console.sh` script:

```sh
PLUGIN_NAME=$(node -r ts-node/register -e "console.log(JSON.stringify(require('./plugin-metadata.ts')))" | jq -r .default.name)
```